batch/batch/driver/billing_manager.py

```diff
@@ -9,40 +10,58 @@
 log = logging.getLogger('billing_manager')
 
 
+ProductVersionInfo = namedtuple('ProductVersionInfo', ['latest_version', 'sku'])
```

Comment:
Can we use a type that encodes that `sku` is an `Optional[str]`? A `dataclass` might be more appropriate here.